web/client/components/resources/modals/Save.jsx

```diff
@@ -122,7 +122,7 @@ class SaveModal extends React.Component {
         canSave: true,
         user: {},
         dialogClassName: '',
-        detailsComponent: require('./enhancers/handleDetails').default((DetailsComp))
+        detailsComponent: require('./enhancers/handleDetails').default(DetailsComp)
```

Comment:
Can we transform this into an import, maybe?

Reply:
yep